7FindFirstandLastElement.py

```diff
@@ -0,0 +1,58 @@
+"""
+LC:34
+#T:O(n)
+"""
+
+#Brute force
+#T:O(n)
+"""
+def getIndex(nums,target):
+  res=[]
+  arr=nums
+  for i in range(len(nums)):
+    j=i+1
+    if arr[i]==target and j<len(arr) and arr[j]==target :
+      res.append(i)
+      res.append(i+1)
+      return res
+  res=[-1,-1]
+  return res
+"""
+#T:O(log n)
+def getIndex(nums,target,l,h):
+  mid=int(l+h/2)
+  result=[]
+  #base
+  while(l<=h):
+   print(nums[l],nums[h],nums[mid])
+   if target==nums[mid]:
+     result.append(mid)
+     previous=mid-1
+     nxt=mid+1
```

Comment:
What if the right most index is 5 elements away from current mid.

Reply:
Will make changes in the code to consider [3,3,3]

Reply:
Made changes in Problem 7-Made first and last index.